lib/cliSuppressExceptions.ts

```diff
@@ -3,9 +3,9 @@
 
 import { log } from "./util/logging"
 
-export const cliSuppressExceptions = async (f: () => Promise<void>): Promise<void> => {
+export const cliSuppressExceptions = async (f: () => Promise<number>): Promise<void> => {
   try {
-    await f()
+    process.exitCode = await f()
```

Comment:
is it [here](https://github.com/Azure/oav/blob/21d90189f5ba347d88784998bda244dde6b3f86c/lib/commands/validate-example.ts#L38) where validateExamples command exit code returns? if so will the exit code propagate out of the "handler" so it's returned as the cli exit code?

Reply:
In this case, `f` is `validateExample`.